apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -286,6 +301,9 @@ class LevelDetailsDialog extends Component {
           <Button
             href={level.url || scriptLevel.url}
             target={'_blank'}
+            onClick={() => {
+              this.recordSeeFullLevelClick(level);
```

Comment:
@bethanyaconnor is level the right thing to pass here or does it depend?

Reply:
Yep this is right! `level` should always be a level here

Reply:
Hm, I guess it depends what you want. Are you looking for the script level id or the level id?

Reply:
Does the script level id just take into account which script the level is accessed through?

Reply:
The level id could be for the same level used in multiple courses. Also tess do you have the lesson id in this?

Reply:
@tess323 Do you want us to record the lesson id with this event?

Reply:
It would be great to capture the lesson id in this as well.